py/desida/fiberassign.py

```diff
@@ -0,0 +1,143 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst.
+"""
+==================
+desida.fiberassign
+==================
+
+Tools for working with *intermediate* fiberassign files, *i.e.* ``${DESI_ROOT}/survey/fiberassign``.
+"""
+import os
+import sys
+import glob
+import shutil
+from argparse import ArgumentParser
+from astropy.io import fits
+from desiutil.log import get_logger, DEBUG
+
+
+log = None
+
+
+def _options():
+    """Parse command-line options.
+
+    Returns
+    -------
+    :class:`argparse.Namespace`
+        The parsed options.
+    """
+    prsr = ArgumentParser(prog=os.path.basename(sys.argv[0]),
+                          description='Create intermediate fiberassign directories.')
```

Comment:
This description string seems incomplete (thinking about whether our future selves will understand the --help output).  I'd suggest something like "move and link intermediate desi/survey/fiberassign files for tiles in a data release".

Reply:
Will do.